content/en/blog/2025/announcing-notation-v2-0-alpha.md

```diff
@@ -76,4 +76,13 @@ To optimize security and performance, this release introduces support for Delta
 
 We encourage you to explore these new features and enhancements by downloading [Notation v2.0.0-alpha.1](https://github.com/notaryproject/notation/releases/tag/v2.0.0-alpha.1) from our GitHub releases page. You can also follow the [installation guidance](/docs/user-guides/installation/cli) for detailed instruction. As this is an alpha release, we welcome your feedback to help us refine and improve the maturity. Please report any issues or suggestions on our GitHub issues page.
 
-Thank you for your continued support and contributions to the Notary Project. Together, we're advancing the security and integrity of software supply chains.
\ No newline at end of file
+Thank you for your continued support and contributions to the Notary Project. Together, we're advancing the security and integrity of software supply chains.
+
+## Join us at KubeCon EU in London
+
+Join two sessions at KubeCon EU in London to explore the challenges, lessons learned, and benefits of using the Notary Project. Dive deep into its mission and strategy, security audit, new use cases, and roadmap.
+
+- [Notary Project: The Key To Secure Software Supply Chain - Yi Zha, Microsoft & Guillaume Gill, OrangeLogic](https://sched.co/1td1W): April 4, 2025 13:45 - 14:15 BST, ICC Capital Suite 14-16, Level 3 
+- [Project Lightning Talk: Notary: Securing Binary Artifacts with Fine-grained Control - Yi Zha, Maintainer](https://sched.co/1tcut): April 1, 2025 10:13 - 10:18 BST, Platinum Suite, Level 3
```

Comment:
Should it be `Notary Project` in the title?

Reply:
Yes, the lightning talk title should also be corrected.